source/basic.tex

```diff
@@ -6277,7 +6276,7 @@
 \begin{example}
 A library I/O function that blocks until the I/O operation is complete can
 be considered to continuously check whether the operation is complete. Each
-such check might consist of one or more execution steps, for example using
+such check possibly consists of one or more execution steps, for example using
```

Comment:
What is the alternative? (If it doesn't consist of one or more execution steps, then what does it do?) Can we drop the "possibly"?

Reply:
dropped